packages/@webex/plugin-cc/src/AgentLogin/AgentLogin.ts

```diff
@@ -0,0 +1,38 @@
+import {HTTP_METHODS, WebexSDK} from '../types';
+
+export default class AgentLogin {
+  webex: WebexSDK;
+  wccAPIURL: string;
+
+  constructor(webex: WebexSDK, wccAPIURL: string) {
+    this.webex = webex;
+    this.wccAPIURL = wccAPIURL;
+  }
+
+  public async loginAgentWithSelectedTeam(
+    teamId: string,
+    agentDeviceType: string,
+    deviceId: string
+  ) {
+    try {
+      const body = {
+        dialNumber: deviceId,
+        teamId,
+        isExtension: agentDeviceType === 'EXTENSION',
+        roles: ['agent'],
+        deviceType: agentDeviceType,
+        deviceId,
+      };
+      const loginResponse = await this.webex.request({
```

Comment:
If we have HttpReq created then why are we using webex.request here directly. Either we should use webex.request everywhere and delete the HttpReq class or use HttpReq for everything